frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -241,9 +243,61 @@ function useCreateHmpbTargetElements({
 const BmdPaperBallotsMemoized = React.memo(BmdPaperBallots);
 const HandMarkedPaperBallotsMemoized = React.memo(HandMarkedPaperBallots);
 
+interface PrintingModalProps {
+  advancePrinting: () => void;
+  currentPrecinct: Precinct;
+  precinctIds: string[];
+  printIndex: PrintIndex;
+}
+
+function PrintingModal({
+  advancePrinting,
+  currentPrecinct,
+  precinctIds,
+  printIndex,
+}: PrintingModalProps): JSX.Element {
+  if (printIndex.component === 'PaperChangeModal') {
+    return (
+      <Modal
+        centerContent
+        content={
+          <Prose textCenter>
+            <p>
+              Load printer with <b>legal-size paper</b>.
+            </p>
+          </Prose>
+        }
+        actions={
+          <Button onPress={advancePrinting}>
+            Legal Paper Loaded. Continue Printing
+          </Button>
+        }
+      />
+    );
+  }
+  return (
+    <Modal
+      centerContent
+      content={
+        <Loading as="p">
+          Printing L&amp;A Package
+          {precinctIds.length > 1
+            ? ` (${printIndex.precinctIndex + 1} of ${precinctIds.length})`
+            : ''}
+          : {currentPrecinct.name}
+        </Loading>
```

Comment:
Question for you @beausmith! What do you think of tweaking the printing modal contents for the all-precincts, legal-size HMPBs case, such that we indicate which portion of the L&A package we're printing, e.g. instead of "Printing L&A Package (1 of 3): District 2", we use "Printing L&A Package (1 of 3): District 2, Letter-Size Contents" / "Printing L&A Package (1 of 3): District 2, Non-Letter-Size Contents"? So the overall printing modal sequence will look as follows:

> Printing L&A Package (1 of 3): District 1, Letter-Size Contents...
> Printing L&A Package (1 of 3): District 2, Letter-Size Contents...
> Printing L&A Package (1 of 3): District 3, Letter-Size Contents...
> Printing L&A Package (1 of 3): District 1, Non-Letter-Size Contents...
> Printing L&A Package (1 of 3): District 2, Non-Letter-Size Contents...
> Printing L&A Package (1 of 3): District 3, Non-Letter-Size Contents...

Reply:
Yes, I love the idea of keeping the user informed with all the data we can offer.

Can we be more specific than "Non-Letter-Size Contents…"?

How about:
- Letter-size pages
- Legal-size pages
- Tabloid-size pages

Reply:
We may even want to change to a multi-line modal:

> **Printing "Central Springfield" L&A Package**
> This is package 1 of 3. 
> Currently printing letter-size pages…

Reply:
This sounds great to me - will update!

Reply:
https://github.com/votingworks/vxsuite/pull/1890/commits/31ef05647f99b505767971a3cf36e9f4f5c61237